packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -1290,7 +1290,9 @@ export class ContainerRuntime
 			config: {
 				compressionOptions,
 				maxBatchSizeInBytes: runtimeOptions.maxBatchSizeInBytes,
-				enableOpReentryCheck: this.enableOpReentryCheck,
+				disablePartialFlush:
+					this.mc.config.getBoolean("Fluid.ContainerRuntime.DisablePartialFlush") ===
+					true,
```

Comment:
unrelated? Seems to wipe out enableOpReentryCheck

Reply:
For the outbox, this is intentional. We won't be checking for op reentry. I mean we'll always do op reentry check but instead of closing the container, we log the instance and (if not disabled) split the batch.

`enableOpReentryCheck` is still in effect for blocking ops from within event handlers. That's caught earlier than attempting to add the op to the batch (via the event handler API on the DDS which will call into the runtime's ensureNoDataModelChanges function)